lnwallet/btcwallet/config.go

```diff
@@ -84,6 +86,14 @@ type Config struct {
 	// DBTimeOut specifies the timeout value to use when opening the wallet
 	// database.
 	DBTimeOut time.Duration
+
+	// RemoteDB should be set the the remote wallet DB when attempting to
+	// use that instead of a locally owned bbolt wallet.db.
+	RemoteDB kvdb.Backend
```

Comment:
See comment re just passing a `kvbd.Backend` here, and lifting the conditional logic to reside one layer up.

Reply:
PTAL